db/migrate/20200615130642_add_bought_cinstances_count_to_accounts.rb

```diff
@@ -0,0 +1,22 @@
+class AddBoughtCinstancesCountToAccounts < ActiveRecord::Migration[5.0]
+  def change
+    safety_assured do
+      add_column :accounts, :bought_cinstances_count, :integer, :null => false, :default => 0
```

Comment:
This is a blocking migration so how do we plan to run this for SaaS?

Reply:
And we should not add anymore column to accounts, this table is terribly huge and is used every time

Reply:
@hallelujah and where you would create that counter attribute?

Reply:
About this: we decided that now 'arbitrarily' splitting the accounts table by this criteria of just being a new attribute will just make the problem worse, so we should add this column to accounts + make a Jira issue to split the accounts table under a well-thought criteria if we end up doing so.